Q/QuEST/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "QuEST"
+version = v"3.7.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/QuEST-Kit/QuEST.git", "d4f75f724993b4af8e43a796e3c09ce24ae11670")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/QuEST
+cmake -B build \
+    -DCMAKE_C_STANDARD=99 \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    .
+cmake --build build --parallel ${nproc}
+mkdir -p "${includedir}"
+cp -vr $WORKSPACE/srcdir/QuEST/QuEST/include/* ${includedir}/
+install -Dvm 755 build/QuEST/libQuEST.${dlext} ${libdir}/libQuEST.${dlext}
+install_license LICENCE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+filter!(platforms) do p
+    !Sys.iswindows(p) &&
+    !(BinaryBuilder.proc_family(p) != "intel" && Sys.islinux(p))
+end
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libQuEST", :quest)
```

Comment:
We usually call the variable with the same name as the corresponding file (all lowercase usually makes things slightly easier): 
```suggestion
    LibraryProduct("libQuEST", :libquest)
```